Makefile

```diff
@@ -127,10 +128,12 @@ fmt:
 vet:
 	$(GO) vet $(PACKAGES)
 
-.PHONY: generate
-generate:
+$(BINDATA_DEST): $(BINDATA_SOURCES) $(JS_SOURCES) $(CSS_SOURCES)
```

Comment:
This is tracking the JS and CSS sources too because I don't think `make` allows to re-define dependencies during runtime.